apidump/apidump.go

```diff
@@ -538,6 +553,13 @@ func (a *apidump) Run() error {
 				return errors.Errorf("Cannot automatically rotate sessions when a session name is provided.")
 			}
 		}
+	} else if args.PostmanCollectionID != "" {
+		args.Out.AkitaURI = &akiuri.URI{
+			ObjectType: akiuri.TRACE.Ptr(),
+			// Placeholder name for postman collectionIds
+			ServiceName: "Postman_" + args.PostmanCollectionID,
```

Comment:
Do we have the real service name we could use?

Reply:
We can get it from the lookup service response. Will raise a new PR for this change